webpack/assets/javascripts/react_app/components/common/AutoComplete/index.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react';
+import Downshift from 'downshift';
+import { Dropdown, MenuItem, InputGroup, FormControl, Button } from 'patternfly-react';
+import PropTypes from 'prop-types';
+
+export const renderItems = ({
+  items, highlightedIndex, selectedItem, getItemProps,
+}) => (
+  <Dropdown.Menu
+    style={{
+      display: 'block',
+      position: 'absolute',
+      left: 'inherit',
+      marginTop: 0,
+      top: 'auto',
+    }}
+  >
+    {items.map((item, index) => (
+      <MenuItem
+        {...getItemProps({
+          index,
+          item,
```

Comment:
in the search auto completer we currently have, we also have a sub section for operators (and, or etc) do you think this component should support it or rather another high level component?

Reply:
This is a really good question. I think we do need to support these sub-sections which means we'll need to support a configuration being passed in rather than just an array of strings. I'll work on adding that in.